### PR TITLE
Handles notifications for legacy projects with no slack channel.

### PR DESCRIPTION
**Pull request by thisisdhaas**

### CODE REVIEW

**Comment by jrbotros** [comment]
lgtm

File: orchestra/utils/notifications.py [hunk 1]
```diff
@@ -84,7 +84,8 @@ def notify_status_change(task, previous_status=None):
         }
 
     _notify_internal_slack_status_change(task, current_worker)
-    _notify_experts_slack_status_change(task, current_worker)
+    if task.project.slack_group_id:
```
Comment (jrbotros) [inline orchestra/utils/notifications.py:87]: Will this error out if the worker doesn't have a slack handle?

**Comment by jrbotros** [comment]
One comment, then lgtm

File: orchestra/utils/notifications.py (hunk 1 above)
Comment (thisisdhaas) [inline orchestra/utils/notifications.py:87]: this line won't--it doesn't use the worker at all. 

Line 132, below, makes sure that there is no @-mention if the worker doesn't have a handle.

Comment (jrbotros) [inline orchestra/utils/notifications.py:87]: Sweet, sounds good.

**Pull request merged**